app/src/main/java/ai/elimu/appstore/synchronization/AppListActivity.java

```diff
@@ -16,55 +16,53 @@
 import ai.elimu.appstore.R;
 import ai.elimu.appstore.dao.ApplicationDao;
 import ai.elimu.appstore.model.Application;
-import timber.log.Timber;
+import ai.elimu.appstore.util.Const;
 
 public class AppListActivity extends AppCompatActivity {
 
-    private TextView textViewLastSynchronization;
+    private TextView mTextViewLastSynchronization;
 
-    private List<Application> applications;
+    private List<Application> mApplications;
 
-    private ArrayAdapter arrayAdapter;
+    private ArrayAdapter mAdapterApps;
 
-    private ListView listViewApplications;
+    private ListView mListViewApps;
 
-    private ApplicationDao applicationDao;
+    private ApplicationDao mApplicationDao;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
-        Timber.i("onCreate");
+//        Timber.i("onCreate");
         super.onCreate(savedInstanceState);
-
         setContentView(R.layout.activity_app_list);
 
-        listViewApplications = findViewById(R.id.listViewApplications);
-
-        textViewLastSynchronization = findViewById(R.id.textViewLastSynchronization);
-
         BaseApplication baseApplication = (BaseApplication) getApplication();
-        applicationDao = baseApplication.getDaoSession().getApplicationDao();
+        mApplicationDao = baseApplication.getDaoSession().getApplicationDao();
+
+        initViews();
     }
 
-    @Override
-    protected void onStart() {
-        Timber.i("onStart");
-        super.onStart();
+    private void initViews() {
+        mListViewApps = findViewById(R.id.listViewApplications);
 
+        mTextViewLastSynchronization = findViewById(R.id.textViewLastSynchronization);
         // Display the time of last synchronization with the server
-        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
-        long timeOfLastSynchronization = sharedPreferences.getLong(AppSynchronizationActivity.DownloadAppListAsyncTask.PREF_LAST_SYNCHRONIZATION, 0);
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences
+                (getApplicationContext());
+        long timeOfLastSynchronization = sharedPreferences.getLong(AppSynchronizationActivity
+                .DownloadAppListAsyncTask.PREF_LAST_SYNCHRONIZATION, 0);
         Date date = new Date(timeOfLastSynchronization);
-        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm");
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(Const.DATE_FORMAT_LAST_SYNC);
```

Comment:
Instead of putting the `DATE_FORMAT_LAST_SYNC` variable in a separate class, add it to the top of the class where it's being used. If it will be used on more places later, we can consider moving it into a common class like this. But for now, try to keep the code loosely coupled.